api/grpcserver/v2alpha1/transaction.go

```diff
@@ -303,7 +304,7 @@ func toTransactionOperations(filter *spacemeshv2alpha1.TransactionRequest) (buil
 	return ops, nil
 }
 
-func toTx(tx *types.MeshTransaction, result *types.TransactionResult,
+func (s *TransactionService) toTx(ctx context.Context, tx *types.MeshTransaction, result *types.TransactionResult,
 	includeResult, includeState bool,
 ) *spacemeshv2alpha1.TransactionResponse {
```

Comment:
NIT: I'd prefer if all arguments were on their own new line 🙂 

```suggestion
func (s *TransactionService) toTx(
	ctx context.Context,
	tx *types.MeshTransaction,
	result *types.TransactionResult,
	includeResult, includeState bool,
) *spacemeshv2alpha1.TransactionResponse {
```